### PR TITLE
Implements additional Room HistoryOptions

### PR DESCRIPTION
**Pull request by afinkenstadt**
Implements #47

### CODE REVIEW

**Comment by afinkenstadt** [comment]
CHeck failed due to absence of `golint`.

File: hipchat/room.go [hunk 1]
```diff
@@ -511,6 +511,14 @@ type HistoryOptions struct {
 	// Reverse the output such that the oldest message is first.
 	// For consistent paging, set to 'false'.
 	Reverse bool `url:"reverse,omitempty"`
+
+	// Either the earliest date to fetch history for the ISO-8601 format string,
+	// or 'null' to disable this filter.
+	// to be effective, API call requires Date also be filled in with an ISO-8601 format string.
```
Comment (tbruyelle) [inline hipchat/room.go]: Do we really need to put the plain string `"null"` ? Or empty is enough ?

Comment (afinkenstadt) [inline hipchat/room.go]: Clarified comment in db6f31b

**Comment by afinkenstadt** [comment]
Closing as it is also included in #49

**Pull request closed without merging**